docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go

```diff
@@ -133,19 +132,14 @@ func (r *CronJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	/*
-
 		<aside class="note">
-
 		<h1>What is this index about?</h1>
-
 		<p>The reconciler fetches all jobs owned by the cronjob for the status. As our number of cronjobs increases,
 		looking these up can become quite slow as we have to filter through all of them. For a more efficient lookup,
 		these jobs will be indexed locally on the controller's name. A jobOwnerKey field is added to the
 		cached job objects. This key references the owning controller and functions as the index. Later in this
 		document we will configure the manager to actually index this field.</p>
-
```

Comment:
We are removing the spaces.
I think that it happens because we are not using a const to inject all those contents

I would suggest we have conts like for example: 

const doc_fragment_x = `

<p>The reconciler fetches all jobs owned by the cronjob for the status. As our number of cronjobs increases,
		looking these up can become quite slow as we have to filter through all of them. For a more efficient lookup,
		these jobs will be indexed locally on the controller's name. A jobOwnerKey field is added to the
		cached job objects. This key references the owning controller and functions as the index. Later in this
		document we will configure the manager to actually index this field.</p>

`

Reply:
It's my bad. I checked and found it is because I missed spaces between lines in a quite long const.
Now returned all spaces.